source/assets/demo/js/toc.js

```diff
@@ -77,15 +77,15 @@
      *
      * @return {Function}
      */
-    var setAttrs = function (overwrite, prefix) {
+    let setAttrs = function (overwrite, prefix) {
         return function ($src, $target, index) {
```

Comment:
(...) => {} ?

Reply:
Refactoring, out of scope, might change behaviour (And i have no ide what toc is :D)